simavr/sim/run_avr.c

```diff
@@ -87,7 +87,7 @@ main(
 		int argc,
 		char *argv[])
 {
-	elf_firmware_t f = {{0}};
+	elf_firmware_t f;
```

Comment:
Why remove the zeroing of this?

Reply:
Because elf_read_firmware() does it.  Leaving it in seems slightly misleading to me as it suggests that you can then put things into the structure and expect them to be used.

But now I have seen issue #400, I can see there is a danger of crashing if --trace is used.  I think the "right thing" is to fix #400 and still remove the initialisation.